cmd/config-v10.go

```diff
@@ -112,6 +112,16 @@ func initConfig() (bool, error) {
 	return false, nil
 }
 
+// reset the value of the Global server config.
+// set it to `nil`.
+func resetGlobalConfig() {
+	// hold the mutex lock before a new config is assigned.
+	serverConfigMu.Lock()
+	// Save the loaded config globally.
+	serverConfig = nil
+	serverConfigMu.Unlock()
+}
+
```

Comment:
Is this function expected to be used in normal functioning of the serve? If not move it to test-utils_test.go.